### PR TITLE
[4.0] Wrong attribute for joomla-tabs ce

### PR DESCRIPTION
**Pull request by dgrammatiko**
Pull Request for Issue #21074 .

### Summary of Changes
Probably another copy paste mistake

### CODE REVIEW

**Comment by brianteeman** [comment]
is the change to the lint file supposed to be here?

**Comment by dgrammatiko** [comment]
nope, will remove it.

Done

File: layouts/joomla/form/field/rules.php [hunk 1]
```diff
@@ -79,7 +79,7 @@
 <p class="rule-desc"><?php echo Text::_('JLIB_RULES_SETTINGS_DESC'); ?></p>
 <?php // Begin tabs ?>
 <joomla-field-permissions class="row mb-2" data-uri="<?php echo $ajaxUri; ?>" id="permissions-sliders">
-	<joomla-tab orientation="vertical" data-ajaxuri="' . $ajaxUri . '" id="permissions-sliders">
+	<joomla-tab orientation="vertical" id="permissions-sliders">
```
Comment (ggppdk) [inline layouts/joomla/form/field/rules.php]: What about also removing or changing duplicate id
`id="permissions-sliders"`
the above line has same id, `<joomla-field-permissions ... id="permissions-sliders">`

Comment (brianteeman) [reply layouts/joomla/form/field/rules.php]: yes that will also cause issues

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**